docker/Dockerfile

```
@@ -20,7 +20,7 @@ RUN gem install bundler -v '~> 1.13' --no-document && \
   gem cleanup
 
 # Install the bundle
-RUN gem install middleman-hashicorp -v ${GEM_VERSION} --no-document
+RUN GEM_PATH=$GEM_HOME gem install middleman-hashicorp -v ${GEM_VERSION} --no-document
```

Style-consistency nit, and I'm a stickler for quoting variables 😄 
```suggestion
RUN GEM_PATH="${GEM_HOME}" gem install middleman-hashicorp -v "${GEM_VERSION}" --no-document
```